src/components/Nav/Nav.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
+import CategoryElement from './CategoryElement';
+import NAV_CATEGORY_LIST from './navCategoryData';
+import USER_CATEGORY_LIST from './userCategoryData';
 import './Nav.scss';
 
 const Nav = () => {
   return (
     <div className="Nav">
-      <div />
+      <div className="navMainCategory">
+        {NAV_CATEGORY_LIST.map(list => (
+          <CategoryElement key={list.id} categoryName={list.categoryName} />
+        ))}
```

Comment:
```suggestion
        {NAV_CATEGORY_LIST.map(({id,categoryName}) => (
          <CategoryElement key={id} categoryName={categoryName} />
        ))}
```
이렇게 구조분해할당 할 수 있습니다.

Reply:
수정 사항 반영해서 구조분해할당 할 수 있는 부분들 수정 했습니다.